packages/backend/src/models/DashboardModel/DashboardModel.ts

```diff
@@ -495,7 +495,6 @@ export class DashboardModel {
                 this.database.raw(
                     `COALESCE(
                         ${DashboardTileChartTableName}.title,
-                        ${SavedChartsTableName}.name,
```

Comment:
This is the fix, if we keep title undefined, everything works as expected. And in the UI we already do `title || chart.name` in most places.